packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -86,8 +86,10 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
   className?: string;
   /**
    * The properties that will get passed to the Suggestions component.
+   *
+   * TODO (adjective-object) remove the '| any' before the next major version
    */
-  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps;
+  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps<T | any>;
```

Comment:
Tied to my comment below, if we don't need to expose this type then this wouldn't actually be a breaking API change.